specification/indices/simulate_template/IndicesSimulateTemplateRequest.ts

```diff
@@ -57,6 +57,10 @@ export interface Request extends RequestBase {
      * @server_default false
      */
     create?: boolean
+    /**
+     * User defined reason for dry-run creating the new template for simulation purposes
+     */
+    cause?: string
```

Comment:
This parameter is read in https://github.com/elastic/elasticsearch/blob/d9e751602dc4a874ef381797844f80044889fbef/server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestSimulateTemplateAction.java#L56 since [7.9.0](https://github.com/elastic/elasticsearch/pull/56842).